packages/block-library/src/columns/editor.scss

```diff
@@ -21,13 +21,34 @@
 
 	> .block-editor-inner-blocks > .block-editor-block-list__layout {
 		display: flex;
-
-		// Responsiveness: Allow wrapping on mobile.
 		flex-wrap: wrap;
+		margin-bottom: $default-block-margin;
 
 		@include break-medium() {
 			flex-wrap: nowrap;
 		}
+
+		@supports (display: grid) {
+			--columns-template: 1fr;
```

Comment:
same comment, might this be better named --grid-template ?

Reply:
I think it's best to keep the explicit reference to columns here, because `grid-template` is an actual CSS property that combines `grid-template-columns` and `grid-template-rows`. I also didn't want the variable to have the exact name of the property because that might be confusing.